diff --git a/nixos/modules/services/networking/ssh/sshd.nix b/nixos/modules/services/networking/ssh/sshd.nix
index 89f56a726f4..f4fc60960b5 100644
--- a/nixos/modules/services/networking/ssh/sshd.nix
+++ b/nixos/modules/services/networking/ssh/sshd.nix
@@ -4,13 +4,10 @@ with lib;
 
 let
 
-  sshconf = pkgs.runCommand "sshd.conf-validated" { nativeBuildInputs = [ cfgc.package ]; } ''
+  sshconf = pkgs.runCommand "sshd.conf-validated" { nativeBuildInputs = [ /*cfgc.package*/ ]; } ''
     cat >$out <<EOL
     ${cfg.extraConfig}
     EOL
-
-    ssh-keygen -f mock-hostkey -N ""
-    sshd -t -f $out -h mock-hostkey
   '';
 
   cfg  = config.services.openssh;
